py3status/modules/usbguard.py

```diff
@@ -1,195 +1,77 @@
-
-format_button_permanent:
-    {permanent}   permanent boolean, eg False, True
-
 format_device:
-    {id}                   eg 1, 2, 5, 6, 7, 22, 23, 33
-    {policy}               eg allow, block
-    {usb_id}               eg 054c:0268
-    {name}                 eg Poker II, PLAYSTATION(R)3 Controller
-    {serial}               eg 0000:00:00.0
-    {port}                 eg usb1, usb2, usb3, 1-1, 4-1.2.1
-    {interface}            eg 00:00:00:00 00:00:00 00:00:00
-    {hash}                 eg ihYz60+8pxZBi/cm+Q/4Ibrsyyzq/iZ9xtMDAh53sng
-    {parent_hash}          eg npSDT1xuEIOSLNt2RT2EbFrE8XRZoV29t1n7kg6GxXg
-    {format_button_allow}  format for allow button
-    {format_button_block}  format for block button
-    {format_button_reject} format for reject button
-
-format_notification_*:
-    See `format_device`
+    {format_button_allow}     button to allow the device
+    {format_button_reject}    button to reject the device
+    {id}                      eg 1, 2, 5, 6, 7, 22, 23, 33
+    {policy}                  eg allow, block, reject
+    {usb_id}                  eg 054c:0268
+    {name}                    eg Poker II, PLAYSTATION(R)3 Controller
+    {serial}                  eg 0000:00:00.0
+    {port}                    eg usb1, usb2, usb3, 1-1, 4-1.2.1
+    {interface}               eg 00:00:00:00 00:00:00 00:00:00
+    {hash}                    eg ihYz60+8pxZBi/cm+Q/4Ibrsyyzq/iZ9xtMDAh53sng
+    {parent_hash}             eg npSDT1xuEIOSLNt2RT2EbFrE8XRZoV29t1n7kg6GxXg
 
 Requires:
     python-gobject: Python Bindings for GLib/GObject/GIO/GTK+
     usbguard: USB device authorization policy framework
 
-Examples:
-```
-# specify a list of filters: None, any, allow, block
-usbguard {
-    * None: show any devices with any options
-    * any: show any devices
-    * allow: show authorized devices
-    * block: show unauthorized devices
-}
-
-# show block device only, hide button
-usbguard {
-    format_button_filter = ''
-    filters = ['block']
-}
-```
-
-@author Cyril Levis (@cyrinux)
+@author @cyrinux, @lasers, @maximbaz
 @license BSD
 
 SAMPLE OUTPUT
 [
-    {'full_text': 'Poker II '},
-    {'full_text': 'Allow ', 'color': '#00ff00'},
-    {'full_text': 'Block ', 'color': '#ffff00'},
-    {'full_text': 'Reject', 'color': '#ff0000'},
-]
-
-filters
-[
-    {'full_text': 'PLAYSTATION(R)3 Controller '},
-    {'full_text': 'Allow ', 'color': '#00ff00'},
-    {'full_text': 'Block', 'color': '#ffff00'},
-]
-
-unknown
-[
-    {'full_text': 'Unknown 12 '},
-    {'full_text': '8087:0024 ', 'color': '#a9a9a9'},
-    {'full_text': 'Allow ', 'color': '#00ff00'},
-    {'full_text': 'Block', 'color': '#ffff00'},
+    {'full_text': '[Reject] ', 'index': '22/reject', 'separator': False, 'separator_block_width': 0, 'urgent': True, 'cached_until': -1},
+    {'full_text': 'USB Flash Drive ', 'separator': False, 'separator_block_width': 0, 'urgent': True},
+    {'full_text': '[Allow]', 'index': '22/allow', 'urgent': True}
```

Comment:
Remove unused keys.  `index`, `separator`, `separator_block_width`.

Reply:
Yeah I was not sure how you generate this comment, I ran `python usbguard.py` and this was printed with default config. Done.

Have you noticed, by the way, that `cached_until` was assigned only to the first object, not to all of them? Is that by design?

Reply:
>Yeah I was not sure how you generate this comment, I ran `python usbguard.py` and this was printed with default config.

Same way. `cached_until` is not needed too. I missed this.

Reply:
Removed, but my question was more about actual behavior of running `python usbguard.py`, is this normal that `cached_until` attribute is added only to the first object instead of all 3?